src/lib/index.ts

```diff
@@ -0,0 +1,169 @@
+import { existsSync, readFileSync } from "fs"
+import merge from "lodash.merge"
+
+const _framework = require("@witnet/solidity").default
+import * as _legacy from "@witnet/sdk/assets"
+import * as _assets from "../../witnet/assets"
+import { readWitnetJsonFiles } from "../helpers.js"
+import * as _utils from "./utils"
+
+export * as utils from "./utils"
+
+export const assets = merge(_legacy, _assets)
+
+export class Rulebook {
+    
+    public readonly priceFeeds;
+    public readonly updateConditions;
+
+    protected _networks;
+    protected _networksPriceFeeds;
+
+    public static default(): Rulebook {
+        const updateConditions = require("../../witnet/updateConditions.json")
+        const priceFeeds = require("../../witnet/priceFeeds.json")
+        return new Rulebook(priceFeeds, updateConditions)
+    }
+
+    public static workspace(): Rulebook {
+        const { priceFeeds, updateConditions } = readWitnetJsonFiles("priceFeeds", "updateConditions")
+        return new Rulebook(priceFeeds, updateConditions)
+    }
+
+    public static fromFilePath(path: string): Rulebook {
+        const { priceFeeds, updateConditions } = Object.fromEntries([
+            "priceFeeds",
+            "updateConditions", 
+        ].map(key => {
+            const filepath = `${path}/${key}.json`
+            if (!existsSync(filepath)) {
+                throw new Error(`Rulebook: file not found: ${filepath}`)
+            } 
+            const rules = readFileSync(filepath, { encoding: "utf-8"})
+            return [ key, existsSync(filepath) ? JSON.parse(rules) : {}]
+        }))
+        return new Rulebook(priceFeeds, updateConditions)
+    }
+
+    public static async fromUrlBase(urlBase: string): Promise<Rulebook> {
+        if (urlBase.startsWith("http")) {
+            return new Rulebook(
+                await fetch(`${urlBase}/priceFeeds.json`), 
+                await fetch(`${urlBase}/updateConditions.json`)
```

Comment:
We can make this calls in parallel instead of waiting for each of them
```suggestion
const [priceFeedsResponse, updateConditionsResponse] = await Promise.all([fetch(priceFeedsUrl), fetch(updateConditionsUrl)])
return new Rulebook(priceFeeds.json(), updateCondition.json())
```